src/sites/twitch-twilight/modules/directory/game.js

```diff
@@ -46,7 +46,7 @@ export default class Game extends SiteModule {
 			for(const inst of instances) this.updateButtons(inst);
 		});
 
-		this.GameHeader.on('update', inst => this.updateButtons(inst));
+		this.GameHeader.on('update', this.updateButtons);
```

Comment:
Even if `updateButtons` doesn't use this, supplying this is always a good idea to prevent any future confusion.